docker-compose.yml

```diff
@@ -790,6 +790,8 @@ services:
       KSQL_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.ksqlDBserver.keystore.jks
       KSQL_SSL_KEYSTORE_PASSWORD: confluent
       KSQL_SSL_KEY_PASSWORD: confluent
+      KSQL_SSL_ENABLED_PROTOCOLS: "TLSv1.3,TLSv1.2"
```

Comment:
Note - this is _required_ for ksqlDB server 6.0.1 to disable TLS1 and 1.1, but is the default/implied in 6.1.0.  Doesn't hurt to set is explicitly anyway.